components/cluster-secret-store-rh/base/appsre-stonesoup-vault-rh-secret-store.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: appsre-stonesoup-vault-rh-secret-store
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  provider:
+    vault:
+      server: "https://vault.ci.ext.devshift.net"
+      path: stonesoup/production/redhat
+      version: v2
+      auth:
+        # VaultAppRole authenticates with Vault using the
+        # App Role auth mechanism
+        # https://www.vaultproject.io/docs/auth/approle
+        appRole:
+          # Path where the App Role authentication backend is mounted
+          path: approle
+          # RoleID configured in the App Role authentication backend
+          roleId: d95dc71a-c0e7-158d-f4e5-75dd5c5c5557
```

Comment:
do we want to hardcode it in the base. Or putting values from overlays make more sense, like the bucket writer?

Reply:
fixed